lib/ical/event.js

```diff
@@ -49,6 +49,14 @@ ICAL.Event = (function() {
     if (options && options.exceptions) {
       options.exceptions.forEach(this.relateException, this);
     }
+
+    if (this.component.parent && !this.isRecurrenceException()) {
```

Comment:
Done by transforming this if in a else if. I also updated the function documentation.